README.md

```diff
@@ -1,56 +1,37 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
-
-
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+# Portfolio
 
+a web page that shows personal and professional information about individual such as skills, projects, contact forms
 
```

Comment:
- [ ] [OPTIONAL] It will be better to add a screenshot for your project to show what you achieved now.